sdk/python/kfp/compiler/pipeline_spec_builder.py

```diff
@@ -1333,3 +1336,33 @@ def get_inputs(task_group: tasks_group.TasksGroup,
     )
 
     return pipeline_spec
+
+
+def write_pipeline_spec_to_file(pipeline_spec: pipeline_spec_pb2.PipelineSpec,
```

Comment:
Is this from some other change? Seems irrelevant to this PR?

Reply:
Yes, good catch. This is duplicate code. I suspect the dupe is attributed to a mistake I made when rebasing this PR.